docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,17 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.10.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.10.0) (03-01-2023)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.10.0...v3.9.0)
+
+### Release Summary
+
+* NEW (CodeAnalyzer): We updated the PMD engine to version 6.54.0.
+* NEW (CodeAnalyzer): We made some updates to the RetireJS vulnerability database.
+* FIX (CodeAnalyzer): We updated the reference to `eslint-recommended` config to point to the updated ESLint dependency. This resolves this issue with new direct or CI/CD installs of Code Analyzer: ```ERROR Cannot find module '/home/runner/.local/share/sfdx/node_modules/eslint/conf/eslint-recommended.js```
```

Comment:
>This resolves this issue

Do we mean this to be:

>This resolves `[this](https://github.com/forcedotcom/sfdx-scanner/issues/986)` issue

Reply:
@rmohan20 Ah, right, it should read: "This resolves the error ....: [error]". I updated this.